docs/design/ovs-pipeline.md

```diff
@@ -193,21 +200,24 @@ This table is used to determine which "category" of traffic (tunnel, local
 gateway or local Pod) the packet belongs to. This is done by matching on the
 ingress port for the packet. The appropriate value is then written to bits
 [0..15] of the NXM_NX_REG0 register: 0 for tunnel, 1 for local gateway and 2 for
-local Pod. This information is used by matches in subsequent tables.
+local Pod. This information is used by matches in subsequent tables. For a
+packet received from the tunnel port, bit 19 of the NXM_NX_REG0 is set to 1, to
+indicate MAC rewritting should be performed for the packet in the
```

Comment:
s/rewritting/rewriting

but I think it should just be "MAC rewrite"

Reply:
Changed to "MAC rewrite".